### DISCUSSION TITLE
Correction to installation command

### DISCUSSION THREAD

**Discussion by b-a-merritt** [Homebrew/discussions#3265] [Linux]
https://docs.brew.sh/Installation

After a few failed installation attempts, I successfully installed by changing the tar option in the command

```Shell
mkdir homebrew && curl -L https://github.com/Homebrew/brew/tarball/master | tar xz --strip 1 -C homebrew
```
to
```Shell
... --strip-components=1 ...
```
which leads me to believe that it is incorrect. [This list of options](https://www.gnu.org/software/tar/manual/html_node/Option-Summary.html) corroborates my idea. I don't think this is PR worthy, but I wanted other users to see it (and possibly have it changed) in the docs.

**Comment by SMillerDev** [comment]
That's probably written for the BSD tar that ships with macOS. If you think it's wrong a pull request would be the best fix.
